tests/artifact/contracts/duplicate_ic_contract_2.py

```
@@ -0,0 +1,22 @@
+# { "Depends": "py-genlayer:test" }
+
+from genlayer import *
```

Ruff (0.11.9)

`from genlayer import *` used; unable to detect undefined names

(F403)